source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/skin.html

```
@@ -15,6 +55,15 @@
     <div class='category altCategoryLine hideWithMenu'></div>
   </div>
 </div>
+
+<!-- Status Bar.   This gets moved to the body -->
+<div class='ca_statusArea'>
+  <span class='caDownloadProgress ca_hide'></span>
+  <span class='caStatusText'></span>
+</div>
+
+</div>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </div> ]

(tag-pair)